src/tests/Logary.Facade.Tests/Facade.fs

```diff
@@ -5,6 +5,23 @@ open Fuchu
 open Logary.Facade
 open ExpectoPatronum
 
+
+type internal TemplateToken =
+    | TextToken of string
```

Comment:
Damn it, this is why I tried to add editor confit at the start, but failed miserably :cry: